build.gradle.kts

```diff
@@ -54,7 +54,7 @@ val preMerge = tasks.register("preMerge") {
 
 plugins {
     id("com.android.library").version("3.5.3").apply(false)
-    id("com.yelp.codegen.plugin").version("1.3.0").apply(false)
+    id("com.yelp.codegen.plugin").version("1.4.0").apply(false)
```

Comment:
Would be possible to reference `PLUGIN_VERSION` here instead of hardcoding the value?

This is not a blocking request but it would be nice if releasing a new version requires one less step :)

Reply:
Looks like we can't as `PLUGIN_VERSION` is inside the `buildSrc` of the `plugin` build and is not accessible from the outer build.